docs/docs-content/user-management/saml-sso/palette-sso-with-entra-id.md

```diff
@@ -0,0 +1,493 @@
+
+This document will guide you on deploying and enabling SSO with Microsoft Entra based on OpenID Connect for integration
+with Palette and Kubernetes clusters.
+
+:::tip
+
+If you want to enable OIDC at the Kubernetes cluster level, refer to the
+[Enable OIDC in Kubernetes Clusters](#enable-oidc-in-kubernetes-clusters) section.
+
+:::
+
+## Enable OIDC SSO in Palette
+
+Use the following steps to enable OIDC SSO in Palette with Microsoft Entra ID.
+
+### Prerequisites
+
+- Palette or Palette VerteX version 4.0.X or greater.
+
+- A [Microsoft Entra ID](https://entra.microsoft.com/#home) subscription. You will need an account with one of the
+  following roles: Global Administrator, Cloud Application Administrator, or Application Administrator. Alternatively,
+  you may be the owner of the service principal.
+
+- If you want to use the same Microsoft Entra ID application for OIDC-based SSO in your Kubernetes cluster itself, you
+  need to install [kubelogin](https://github.com/int128/kubelogin) on your local workstation to handle retrieval of
+  access tokens for your cluster.
+
+### Configure Microsoft Entra ID with Palette
+
+1.  Log in to [Palette](https://console.spectrocloud.com) as a **Tenant Admin**.
+
+2.  Navigate to the left **Main Menu** and select **Tenant Settings**. From the **Tenant Menu**, select **SSO**, then
+    **Configure**, and lastly, click on the **OIDC** tab.
+
+    ![A view of the callback URL field](/oidc-entra-id-images/oidcallback.png)
+
+3.  Copy the **Callback URL** to your clipboard. This URL will be used in the next step to configure Microsoft Entra ID.
+
+4.  In a separate browser tab, log in to the Microsoft Entra ID Admin console and open the
+    [App registration blade](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade).
+
+5.  Click on **New registration** and assign a name to the new registration. In this guide, the example app registration
+    is named _palette_oidc_.
+
+6.  Next, select **Web**, and paste the _Palette Callback URL_ value from your clipboard into the **Redirect URL
+    (optional)** field, and Click **Register**.
+
+    ![Azure application creation screen](/oidc-entra-id-images/palette-registration.png)
+
+7.  From the app overview page, navigate to the left **Main Menu** and select **Certificates & secrets**. In the
+    following screen, click on **New client secret**.
+
+8.  Add a description for the secret and select an expiration period. Click on **Add** to create the secret.
+
+    :::warning
+
+    We recommend you store the secret value in a secure location, such as a password manager. You will need this value
+    later when you configure Palette.
+
+    :::
+
+9.  From the application overview page, navigate to the left **Main Menu** and select **Token configuration**.
+
+10. Select **Add optional claim** button. Choose **Token type** as **ID**, and add the claims **email** and
```

Comment:
To support Entra v2 tokens, you'll also need to add the following optional claims: `family_name` and `given_name`.